packages/client-generator-ts/src/TSClient/file-generators/ClientFile.ts

```diff
@@ -29,6 +29,7 @@ export function createClientFile(context: GenerateContext, options: TSClientOpti
 
   const exports = [
     ts.moduleExportFrom(context.importFileName('./enums')).asNamespace('$Enums'),
```

Comment:
Is this still necessary?

Reply:
It mirrors things from `prisma-client-js`, actually